physionet-django/console/views.py

```diff
@@ -2885,9 +2886,40 @@ def event_management(request, event_slug):
     Admin page for managing an individual Event.
     """
     selected_event = get_object_or_404(Event, slug=event_slug)
+
+    event_dataset_form = EventDatasetForm()
+
+    # handle the add dataset form(s)
+    if request.method == "POST":
+        if 'add-event-dataset' in request.POST.keys():
+            event_dataset_form = EventDatasetForm(request.POST)
+            if event_dataset_form.is_valid():
+                if selected_event.datasets.filter(
+                        dataset=event_dataset_form.cleaned_data['dataset'],
+                        status=EventDataset.EventDatasetStatus.Active).count() == 0:
```

Comment:
This works for our immediate aim but isn't future proof. We'll probably want the ability add datasets through multiple access mechanisms, e.g. you are allowed to use this dataset via BigQuery and Google Storage (GBQ / GCS). This will only allow us to add a dataset for a single access mechanism. I'd change this to check against access type as well.

Reply:
Good point. I originally did the code to check against access type. but reverted it to this(just in case someone might suggest that we just want the single access mechanism)

Let me fix this.